### PR TITLE
Create an ADR for links-based facets

### PR DESCRIPTION
**Pull request by leenagupte**
Trello: https://trello.com/c/FrKLI3vc

Rendered version [here](https://github.com/alphagov/content-tagger/blob/3b1619f4ad3d69b4641d5a4308f8e253bf31d5a8/docs/arch/adr-001-links-based-facets.md)

### CODE REVIEW

**Comment by vanitabarrett** [comment]
From my limited memory of the business finder, this looks ok and seems to cover everything 👍

File: docs/arch/adr-001-links-based-facets.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+but the results between the rest of the facets was "additive". Though this has since been changed to have an "additive" relationship between all of the facets.
+
+Normally facets are defined in the "details" hash of the content item. "details" are updated every time the content item is published via the "put_content" method in publishing-api. After being published, the content item is reindexed in search with this facet information.
+
+This meant that to be able to tag content to facets in the business finder, an interface would need to be added to every publishing app.
+
+The quick solution to this problem was to bypass the publishing pipeline completely, and instead update the facet information in search directly.
+
+Simply put, content designers would update a CSV file with a list of content and the business finder facets it needed to be tagged to. Developers would upload this CSV file and then run a script to tag the content in search.
+The script would check whether it already knew about the tags, and if it didn't, would add them. It also triggered an email every time a tag was added.
```
Comment (steventux) [inline docs/arch/adr-001-links-based-facets.md]: I think this should be _It triggered an email every time a new item of content was added to the business finder._

File: docs/arch/adr-001-links-based-facets.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+This manual process was run twice each day.
+
+### The problem
+
+Content items aren't just re-published when they're updated, but also when the things they are linked to are updated, e.g. an organisation, step by step or taxon.
+
+As mentioned before, when content is updated, it is reindexed in search. That means that any business finder related tags are over written ("fall off" the search index) until the script is run again and they are re-added.
+This means that in the time between the content being updated and the script being run, the content does not appear in the finder.
+
+Another more serious problem was that, because an email is triggered every time the tags are added to search, you could (and did) end up in a situation where it looked like a piece of content was being updated twice a day, when in reality only the tags, or related content was being updated.
```
Comment (steventux) [inline docs/arch/adr-001-links-based-facets.md]: See the earlier comment, it's more accurate to say an email was triggered when a new content item was introduced to the finder.

**Comment by leenagupte** [comment]
@steventux I've made the changes and fixed up. 🎉

**Pull request merged**